tests/x11/systemsettings5.pm

```diff
@@ -0,0 +1,28 @@
+# SUSE's openQA tests
+#
+# Copyright © 2009-2013 Bernhard M. Wiedemann
+# Copyright © 2012-2016 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+use base "x11test";
+use testapi;
+
+sub run() {
+    my $self = shift;
+    x11_start_program("systemsettings5", 6, {valid => 1});
+    if (get_var("LIVETEST")) {
+        assert_screen 'test-systemsettings-1', 15;
```

Comment:
You know, as @coolo pointed in some other PR. When people start copy-pasting existing code IMO its better with defaults rather then arbitrary options valid only for some distant test.

Personally I think for `assert_screen` always use default unless you need to wait longer. Shorter timeout will not speed up the test.
For `check_screen` that's different and I see there are use cases for shorter timeouts and ideally there should be accompanying comment explaining why shorter timeout (\me going full @okurz ) :)